cli.js

```diff
@@ -47,7 +47,14 @@ const options = require("nomnom")
         abbr: "b",
         flag: true,
         default: false,
-        help: "Restores the old behavior of (pre-0.8.0) KaTeX.",
+        help: "Restores the old (pre-0.8.0) behavior of \\color.",
```

Comment:
I think describing its behavior, i.e., `\color` will work like LaTeX's `\textcolor` is better.

Reply:
Sounds good; pushed a revision.